### ISSUE TITLE
Clicking the window tab with the middle mouse button opens up a new browser tab

### ISSUE THREAD

**Issue by filippsen** [SuperblocksHQ/superblocks-lab#60]
## Actual result
The window tab refers to the windows opened in the main editor area e.g. when opening a contract file for editing. Clicking HelloWorld.sol tab with the middle mouse button would cause a new browser tab to open with another instance of Studio.

## Expected result
Nothing

**Comment by filippsen** [comment]
There is currently no handler for middle-mouse button actions. Also, it relates to browser specific behavior. Given that, some "references" might consider this as "working as intended".

Adding a new event handler (for example `onMouseDown`) to `renderHeader` in `src/components/projecteditor/panes.js` makes it possible to read the middle-mouse button action, but it is not enough to prevent _Chrome_ or _Firefox_ from opening a new tab.

Reference:  
```
<a href="#" className={classnames(cls)} onMouseDown={(e) => this.tabClickedMiddle(e)} onClick={(e) => this.tabClicked(e, pane.id)} onContextMenu={(e)   => this.tabRightClicked(e, pane.id)}>
tabClickedMiddle = (e) => {
    if(e.button == 1) {
        e.preventDefault();                                                                                                                                           
        e.stopPropagation();  // Important so we don't trigger tabClicked
    }   
}  
```

Unable to intercept action on the spot. Any ideas ? Is it relevant ?

**Comment by javier-tarazaga** [comment]
@filippsen I think the reason might be with the fact that we are using <a> instead of an actual button for clickable items. Have you seen this behavior in a <button> component?

**Comment by filippsen** [comment]
@javier-tarazaga The only other I place I spotted the same behavior is in the lightning/thunder icon (recompile or redeploy action, at the top left side of the respective windows). That one also happens to be an anchor element.

**Comment by filippsen** [comment]
Complementing: removing the `href="#"` would solve the issue. However, it would require an addition to set the cursor pointer style back the way it was when the file tab was a link (on mouse over).

**Comment by javier-tarazaga** [comment]
Solved in #210
